benchmarks/benchmarks/optimize_milp.py

```diff
@@ -0,0 +1,71 @@
+import os
+
+import numpy as np
+from numpy.testing import assert_allclose
+
+from .common import Benchmark, safe_import
+
+with safe_import():
+    from scipy.optimize import milp
+
+with safe_import():
+    from scipy.optimize.tests.test_linprog import magic_square
+
+
+# MIPLIB 2017 benchmarks included with permission of the authors
+milp_problems = ["piperout-27"]
```

Comment:
I don't see a script or explanation for how the `.npz` file in this PR was created. That would be useful to add.

Reply:
@mckib2 converted the problem from `mps` format to arrays using his `scikit-glpk`, then we saved them to `.npz`. Do you want that stated here? e.g.:
> Loaded from original MPS format using scikit-glpk

Reply:
I suspect if anything needs to be done here, it's just a hint that an (any) MPS converter was used to load the problem into Python and then rearranged in the linprog-style and saved back out to `npz`

Reply:
```suggestion
# The MIPLIB benchmark problem set was downloaded from https://miplib.zib.de/
# An MPS converter (scikit-glpk) was used to load the problem into Python. The
# arrays were arranged to the format required by `milp` and saved to `npz`
# format using `np.savez`.
milp_problems = ["piperout-27"]
```

Reply:
Hinted.